docs/basics/101-116-sharelocal.rst

```diff
@@ -52,7 +52,7 @@ how a dataset can be obtained from a path (instead of a URL as shown in the sect
 :ref:`installds`). Thirdly, ``DataLad-101`` is a dataset that can
 showcase many different properties of a dataset already, but it will
 be an additional learning experience to see how the different parts
-of the dataset -- text files, larger files, datalad subdataset,
+of the dataset -- text files, larger files, :dlcmd:`subdataset`,
```

Comment:
Shouldn't this say "subdatasets" and not be a command?

Reply:
The sentence is complicated. This was my interpretation.

Reply:
Updated